tests/functional/nix-profile.sh

```diff
@@ -58,7 +58,7 @@ nix profile list | grep -A4 'Name:.*flake1' | grep 'Locked flake URL:.*narHash'
 [ -e $TEST_HOME/.nix-profile/share/man ]
 (! [ -e $TEST_HOME/.nix-profile/include ])
 nix profile history
-nix profile history | grep "packages.$system.default: 1.0 added"
+nix profile history | grep "packages.$system.default: 1.0, 1.0-man added"
```

Comment:
`1.0-man` seems wrong to me. "man" should be parsed as an output name, not a version. I don't see anything in the PR that would cause this change though...

Reply:
Yeah, the old version matched because `1.0` was at the end of the match. It's been printing the `1.0, 1.0-man` for "some time" because of the loose match